owlkettle/gtk.nim

```diff
@@ -87,6 +87,23 @@ type
     GTK_SHADOW_ETCHED_IN,
     GTK_SHADOW_ETCHED_OUT
   
+  GtkStateFlag* = enum
```

Comment:
This isn't explicitly needed anywhere, I thought I'd need it because of GtkParamSpec IIRC, but it ended up being irrelevant.
I'd keep it here simply so that we have the enum bound should we need it in the future.
Of course if it is more desired to keep the bindings as minimal as they can be then this enum should be removed

Reply:
If it is not needed, it is probably cleaner to remove it.